www/components/poll-sampler.php

```diff
@@ -25,115 +25,58 @@
                 sandbox in (0, ?)
               group by
                 p.pollid";
-$limit = "limit 0, 5";
 
-$rows1 = [];
-$rows2 = [];
-$rows3 = [];
+$recently_created = [];
+$recently_voted = [];
 
 // query the most recently created polls
 $result = mysqli_execute_query($db,
-    "$baseQuery order by p.created desc $limit", [$sandbox]);
-while ($row = mysql_fetch_row($result))
-    $rows1[] = $row;
+    "$baseQuery order by p.created desc limit 0, 5", [$sandbox]);
+$recently_created = mysqli_fetch_all($result);
 
 // add the most recently active polls (i.e., latest vote)
 $result = mysqli_execute_query($db,
-    "$baseQuery order by lastvotedate desc $limit", [$sandbox]);
-while ($row = mysql_fetch_row($result))
-    $rows2[] = $row;
-
-// add the most popular polls (most votes)
-$result = mysqli_execute_query($db,
-    "$baseQuery order by votecnt desc, gamecnt desc $limit", [$sandbox]);
-while ($row = mysql_fetch_row($result))
-    $rows3[] = $row;
-
-function addPollRows(&$dst, &$src, $allRows, $amount)
-{
-    foreach (range(1, $amount) as $i) {
-        // add the next element of src not in dst
-        if (addUniquePollRow($dst, $src))
-            continue;
-
-        // if that failed, fall back to the master list
-        addUniquePollRow($dst, $allRows);
-    }
+    "$baseQuery order by lastvotedate desc limit 0, 5", [$sandbox]);
+$recently_voted = mysqli_fetch_all($result);
+
+echo "<div class=\"block\"><div class=\"headline\">Vote!</div>"
+    . "<p>Help other IFDB members find the games they're looking for "
+    . "by voting in their polls.  Here are a few recent ones:</p>";
+
+global $nonce;
+echo "<style nonce='$nonce'>\n"
+    . ".poll-sampler__create { margin-top: 1ex; }\n"
+    . "</style>\n";
+
+function displayPoll($row) {
+    [$pollID, $pollTitle, $pollDesc, $pollDate, $pollUserID,
```

Comment:
unused fields that can be removed from the select:
`$pollDesc`, `$pollDate`, `$pollVotes`, `$pollGames`, `$pollLastVoteDate`, `$pollUserName`.

Reply:
Done